app/src/components/misc/SignInViaEmailLink/index.css

```diff
@@ -0,0 +1,14 @@
+.form-input {
+  box-shadow: var(--input-box-shadow);
+  border: 0 !important;
+  min-height: 38px;
+  width: 80%;
+}
+
+.container {
+  margin: 20px 50px;
+  padding: 0 50px;
+}
+.mb-20 {
+  margin-bottom: 20px;
+}
```

Comment:
- These class names are generic. If you require class for a specific component, please use specific class names for the component.

- `mb-20` can be setup in `base.css` if you require.
- Avoid use of `!important`